xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/VideoLayerBridgeDRMPRIME.cpp

```diff
@@ -31,6 +31,18 @@ void CVideoLayerBridgeDRMPRIME::Disable()
   struct plane* plane = m_DRM->GetVideoPlane();
   m_DRM->AddProperty(plane, "FB_ID", 0);
   m_DRM->AddProperty(plane, "CRTC_ID", 0);
+
+  // disable HDR metadata
+  struct connector* connector = m_DRM->GetConnector();
+  if (m_DRM->SupportsProperty(connector, "HDR_OUTPUT_METADATA"))
+  {
+    m_DRM->AddProperty(connector, "HDR_OUTPUT_METADATA", 0);
+    m_DRM->SetActive(true);
+
+    if (m_hdr_blob_id)
+      drmModeDestroyPropertyBlob(m_DRM->GetFileDescriptor(), m_hdr_blob_id);
```

Comment:
We probably need to do the same in the destructor?

Reply:
Under normal circumstances the video player bridge should always get a call to `Disable` before being destructed, see https://github.com/xbmc/xbmc/blob/master/xbmc/windowing/gbm/WinSystemGbm.cpp#L204-L228

I guess during a crash or possible when the video layer is active when Kodi is exiting `Disable` may not be called. I will investigate and possible add proper Disable/reset to the gbm window system deinit process in a separate PR if needed.

Reply:
Okay, looks good!